### DISCUSSION TITLE
Custom Video Projection surface

### DISCUSSION THREAD

**Discussion by snovak** [mozilla/hubs#3987] [Spoke Questions] [answered]
Is it possible to define a custom geometry on which to play a video?  Besides "flat" and "360-equitrectangular", it would be great if I could plug in a UUID from another object in that field ;-)

Can I create/import a custom .glb and define a video as the texture somehow?  I imagine you lose all the controls and extra goodies, but if it could be defined to simply loop, I'd be happy.  

Is there anyway to make this work?

**Comment by johnshaughnessy** [comment, marked as answer]
Currently, there's no mechanism to have video playback on arbitrary geometry. I agree though that it sounds like a great feature. It may be something we build in the future (likely as part of the blender plugin for hubs https://github.com/MozillaReality/hubs-blender-exporter )

**Comment by vincentfretin** [comment]
I just found back this tweet for you, someone did a Custom Video Geometry for Hubs, see
https://twitter.com/utopiah/status/1333528657923624960
